setup.py

```diff
@@ -166,6 +166,8 @@ def find_version(*file_paths):
         'typing-extensions',  # backwards-compatibility for newer typing constructs
         'opentelemetry-api',
         'tblib',
+        'tomli',
+        'sparseconverter',
```

Comment:
Should have a lower constraint `>=0.3.0` probably

Reply:
👍